app/src/main/res/layout/app_bar_chat_rooms.xml

```diff
@@ -23,9 +23,7 @@
             android:drawablePadding="10dp"
             android:fontFamily="sans-serif-medium"
             android:maxLines="1"
-            android:textColor="#FFFFFF"
-            android:textSize="20sp"
-            android:textStyle="normal"
+            style="@style/TextAppearance.AppCompat.Widget.ActionBar.Title"
```

Comment:
@filipedelimabrito This is default style of the toolbar title. Having this style also helps us to have different style and font on the title on landscape and portrait mode. otherwise, in landscape, the title font will be the same(20sp) and will look quite big.